docs/README.md

```diff
@@ -49,19 +49,19 @@ virtual environment: run `pipenv install` again in the root folder of the reposi
 
 1. Navigate to the root folder of the repository.
 
-2. Start a local web server on `http://localhost:8000/` by running `./docs/scripts/preview.sh`. In case an
-   old/cached html document is being shown, make sure to refresh your web browser.
+2. Start a local web server on `http://localhost:8000/` by running `./docs/scripts/preview.sh`. In
```

Comment:
I think this part comes from some formatting settings you have in your VS code. I think it would be good to share these settings across the team, so that our commits do not generally include reformattings. 

Happy to apply the same settings you are using myself

Reply:
actually, I did that manually to get less than 100 characters (like the other lines in the file). maybe we should agree upon 80 or 100 characters per line in general (for the style guide). there is a setting in visual studio which helps visualize this: `"editor.rulers": [80, 100]`

Reply:
Agree, let's pick it up next team meeting?